core/consensus/grandpa/impl/vote_tracker_impl.hpp

```diff
@@ -15,20 +15,26 @@ namespace kagome::consensus::grandpa {
    public:
     using PushResult = typename VoteTracker<MessageType>::PushResult;
     using VotingMessage = typename VoteTracker<MessageType>::VotingMessage;
+    using EquivocatoryVotingMessage =
+        typename VoteTracker<MessageType>::EquivocatoryVotingMessage;
+    using VoteVariant = typename VoteTracker<MessageType>::VoteVariant;
 
     ~VoteTrackerImpl() override = default;
 
     PushResult push(const VotingMessage &vote, size_t weight) override;
 
-    std::vector<VotingMessage> getMessages() const override;
+    std::vector<VoteVariant> getMessages() const override;
 
-    size_t getTotalWeight() const override;
+    size_t totalWeight() const override;
```

Comment:
Why rename it? Even here the above method is 'getMessages', looks a bit inconsistent.